examples/plot_fmri_data_example.py

```diff
@@ -129,55 +121,22 @@ def preprocess_haxby(subject=2, memory=None):
 # --------------------------------------------
 
 #############################################################################
-# First, we try to recover the discriminative partern by computing
-# p-values from SVR decoder weights and a parametric approximation
-# of the distribution of these weights.
-beta_hat, scale = empirical_thresholding(
-    X,
-    y,
-    linear_estimator=LinearSVR(),
-)
-pval_std_svr, _, one_minus_pval_std_svr, _ = pval_from_scale(beta_hat, scale)
-
-#############################################################################
-# Now, we compute p-values thanks to permutation tests applied to
-# 1/the weights of the SVR decoder or 2/the weights of the Ridge decoder.
-
-# To derive the p-values from the SVR decoder, you may change the next line by
-# `SVR_permutation_test_inference = True`. It should take around 15 minutes.
-
-SVR_permutation_test_inference = False
-if SVR_permutation_test_inference:
-    # It will be better to associate cross validation with the estimator
-    # but for a sake of time, this is not done.
-    estimator = LinearSVR()
-    weight_svr, weight_svr_distribution = permutation_test(
-        X, y, estimator, n_permutations=50
+# First, we try to recover the discriminative parttern by computing
+# p-values from desparsified lasso.
+# Due to the size of the X, it's not possible to use this method with a limit
+# of 5 G for memory. To handle this problem, the following methods use some
+# feature aggregation methods.
+try:
+    beta_hat, sigma_hat, precision_diagonal = desparsified_lasso(
+        X, y, noise_method="median", max_iteration=1000
     )
-    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_pval(
-        weight_svr, weight_svr_distribution
+    pval_dl, _, one_minus_pval_dl, _, cb_min, cb_max = desparsified_lasso_pvalue(
```

Comment:
Why are you adding the desparsified lasso here ? It is horribly expensive and powerless.
you actually want to show users that this is a bad thing to do ? I find it a bit weird.

Reply:
I want to show that it's not possible to Desparsified Lasso alone but it requires Clustering Inference or Ensemble Clustering Inference.

One reason is that DL requires too many computational resources (memory in this case).

Reply:
But if the tests are run on  a super-server, this might fit in memory... and it would take ages !
You probably want to show the poor scaling of FL on a much smaller example. Another possibility is to compare (En)CluDL with different numbers of clusters and show that increasing it leads to more computation and lower power.

Reply:
It's not possible because I added a limiter of used resources. The only problem is to run the example on a computer which has less than 6 GB of Ram.